### ISSUE TITLE
[BUG]: Modal not large enough by default

### ISSUE THREAD

**Issue by QuentiumYT** [neuromorph/openbar#98]
**Describe the bug**
When opening the extension's settings, the modal width is not large enough, requiring to always enlarge it or make it full screen. Can it be adjusted so it'll fit better? It should be like 1000px wide at least.

**Relevant  Specs:**
 - Open Bar version: 42
 - Gnome version: 47.0
 - OS/Distribution: Ubuntu 24.10

**Screenshots**
![image](https://github.com/user-attachments/assets/acc577c8-3fec-4681-925b-2567e95f6b52)
![image](https://github.com/user-attachments/assets/e866585a-3b9d-45c4-860a-ad91e5d73e6f)

**Comment by neuromorph** [comment]
Oops, that's a bug. A change was made in the latest version to accommodate for low resolution screens and it seems to have broken it.

Please let me know your screen resolution and display scaling factor. 

Thanks for reporting!

**Comment by QuentiumYT** [comment]
Oh alright, yeah I saw the code, smart thing to adapt to the screen resolution.

I have a laptop with a 2560x1600px with a scale factor of 100%.
Also, I have the accessibility setting "Large text" that makes it like twice bigger, which correspond to the default text scale of Ubuntu.

Don't hesitate to ask me to test the extension before a release or if you are not sure it will fit, a large resolution is not so common I guess.

**Comment by neuromorph** [comment]
> Also, I have the accessibility setting "Large text" that makes it like twice bigger, 

This is probably causing the issue. 

Your display is high-res but it also seems to be the trend now. On the other hand there is another issue opened in parallel where they are facing issue due to low res display of 768px :) , so it is a tug of war :p 

I had kept it Gtk only to support Gnome 42 but may have to rewrite into Adwaita which is more responsive. However, that is considerable effort for which I don't have the bandwidth currently. I will see what can be done in a more immediate timescale.

Meanwhile, feel free to increase the width in `prefs.js` for your local install to avoid text getting cut.

**Comment by QuentiumYT** [comment]
Alright, looks like it was this issue. I explored a little bit because I did some tests with another screen as well and I just reset the scale factor to 1 (1.01 actually, else Ubuntu set it back to 1.25) and increased only the built-in screen to 125%

![image](https://github.com/user-attachments/assets/0b2a8471-bc87-4a8e-a63f-da59329dab48)
I had to enable Fractional Scaling tho for people interested. So the text is no longer bigger on every screen now which is more logic as well :) Thanks for the investigation
